slides/04_modules.md

```diff
@@ -37,25 +37,37 @@ Unported License, [http://creativecommons.org/licenses/by-sa/4.0/](http://creati
 # How to use modules (1/2)
 
 - The syntax is simple: `module command module-name`
-    - For example: `module load gromacs-env`
-    - [List of most common commands](https://docs.csc.fi/computing/modules/#module-commands-table)
+   - For example: `module load gromacs-env`
+   - [List of most common commands](https://docs.csc.fi/computing/modules/#module-commands-table)
 - You can't just load all the modules because of the dependencies
 - If you try to load a module that is not available, you will get an error message saying so 
 
 # How to use modules (2/2)
- - These commands will help you figuring out the module situation:
-    - `module list`: See the list of modules loaded at the moment
-    - `module avail`: Modules available at the moment (due to depencies -hides modules that can't be loaded atm)
-     - `module spider name`: Search for an application in the list of all existing modules
-     - `module spider name/version`: Gives information on how to load the module (prerequisites etc).
 
-# Conda environments
+- These commands will help you figuring out the module situation:
+   - `module list`: See the list of modules loaded at the moment
+   - `module avail`: Modules available at the moment (due to depencies -hides modules that can't be loaded atm)
+   - `module spider name`: Search for an application in the list of all existing modules
+   - `module spider name/version`: Gives information on how to load the module (prerequisites etc).
+
+# Various kinds of modules
+
+- Some modules contain software that has a graphical user interface
+   - For example chemistry visualisation software [Molden](https://docs.csc.fi/apps/molden/)
+   - Use Puhti Web Interface or [other connection with GUI](https://docs.csc.fi/computing/connecting/#using-graphical-applications)
+- Some modules contain software that provide a command line interface
+   - For example [Allas object storage](https://docs.csc.fi/data/Allas/)
```

Comment:
```suggestion
   - For example [Allas module](https://docs.csc.fi/data/Allas/accessing_allas/#accessing-allas-in-the-csc-computing-environment-and-other-linux-platforms) which provides tools to access the object storage 'Allas'
```

Reply:
Yeah I tried to come up with a good example but don't know the collection so well :D

Reply:
```suggestion
   - For example [zstd module](https://docs.csc.fi/support/tutorials/env-guide/packing-and-compression-tools/#zstandard-compression-tool)
```